_tutorials/new-component/create-a-new-component-part-1.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Create A New Origami Component
+description: A step-by-step tutorial which teaches you how to build and deploy a new Origami component.
+cta: Learn how to create an Origami component
+
+---
+
+# {{page.title}}
+
+The "Create A New Origami Component" tutorial is split into eight parts and is intended to be followed sequentially from start to finish:
+- part one: Intro & Boilerplate
+- [part two](/docs/tutorials/create-a-new-component-part-2/): Base Styles
+- [part three](/docs/tutorials/create-a-new-component-part-3/): Themes & Brands
+- [part four](/docs/tutorials/create-a-new-component-part-4/): Demos
+- [part five](/docs/tutorials/create-a-new-component-part-5/): JavaScript
+- [part six](/docs/tutorials/create-a-new-component-part-6/): Testing
+- [part seven](/docs/tutorials/create-a-new-component-part-7/): Documentation
+- [part eight](/docs/tutorials/create-a-new-component-part-8/): Component Lifecycle
+
```

Comment:
What do we think to this being an ordered list?
Before:
<img width="555" alt="image" src="https://user-images.githubusercontent.com/1569131/87307281-33a83900-c511-11ea-9b96-d52458d9138b.png">

After:
<img width="626" alt="image" src="https://user-images.githubusercontent.com/1569131/87307261-28eda400-c511-11ea-96df-3a56d796a260.png">

```suggestion
-

1. Intro & Boilerplate
2. [Base Styles](/docs/tutorials/create-a-new-component-part-2/)
3. [Themes & Brands](/docs/tutorials/create-a-new-component-part-3/)
4. [Demos](/docs/tutorials/create-a-new-component-part-4/)
5. [JavaScript](/docs/tutorials/create-a-new-component-part-5/)
6. [Testing](/docs/tutorials/create-a-new-component-part-6/)
7. [Documentation](/docs/tutorials/create-a-new-component-part-7/)
8. [Component Lifecycle](/docs/tutorials/create-a-new-component-part-8/)

```